### ISSUE TITLE
Chef-server-ctl doesn't fail without sudo/root  UX

### ISSUE THREAD

**Issue by kagarmoe** [chef/chef-server#1597]
NOTE:
We use GitHub issues to track bugs and feature requests. If you need help please post to our Mailing List or join the Chef Community Slack.

 * Chef Community Slack at http://community-slack.chef.io/.
 * Chef Mailing List https://discourse.chef.io/

### Chef Server Version
12.18.14 (Current)

### Platform Details
Ubuntu 18.4 Server instance in Azure.  It is a Standard D2s_v3, which has 8 gb memory and 16gb SSD temp storage

### Configuration
Standalone

### Scenario:
Installing the standalone server, chef-server-ctl commands don't fail clearly when run without sudo. 

```
azure@chefserver1128:~$ chef-server-ctl list-server-admins
Traceback (most recent call last):
        9: from /usr/bin/chef-server-ctl:109:in `<main>'
        8: from /usr/bin/chef-server-ctl:109:in `load'
        7: from /opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/chef-server-ctl-1.0.0/bin/chef-server-ctl:254:in `<top (required)>'
        6: from /opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/omnibus-ctl-0.6.0/lib/omnibus-ctl.rb:747:in `run'
        5: from /opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/omnibus-ctl-0.6.0/lib/omnibus-ctl.rb:204:in `block in add_command_under_category'
        4: from /opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/chef-server-ctl-1.0.0/plugins/server_admins.rb:123:in `block in load_file'
        3: from /opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/omnibus-ctl-0.6.0/lib/omnibus-ctl.rb:478:in `running_service_config'
        2: from /opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/omnibus-ctl-0.6.0/lib/omnibus-ctl.rb:466:in `running_package_config'
        1: from /opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/omnibus-ctl-0.6.0/lib/omnibus-ctl.rb:444:in `running_config'
/opt/opscode/embedded/lib/ruby/gems/2.5.0/gems/omnibus-ctl-0.6.0/lib/omnibus-ctl.rb:444:in `read': Permission denied @ rb_sysopen - /etc/opscode/chef-server-running.json (Errno::EACCES)
azure@chefserver1128:~$
```

**Comment by stevendanna** [comment]
@markan Yup, we used to have such a check in the old shell wrapper, but I think we accidentally dropped it when moving to the binliked executable: https://github.com/chef/chef-server/commit/70a2b34a3fcbcfed4bbe4015d16d74bb1cef013a#diff-4b5b0ffe9dd4f6ff720f46ea17300d0bL67

**Comment by PrajaktaPurohit** [comment]
Fixed in 14.0.58. Closing.
